main.py

```diff
@@ -0,0 +1,314 @@
+ed_sheeran = song_titles["EdSheeran"]
+adele = song_titles["Adele"]
+beyonce = song_titles["Beyonce"]
+george = song_titles["GeorgeStrait"]
+johnny_cash = song_titles["JohnnyCash"]
+garth_brooks = song_titles["GarthBrooks"]
+carrie_underwood = song_titles["CarrieUnderwood"]
+
+kpops_factory = CallbackData('kpops_id', prefix='kpops')
+rocks_factory = CallbackData('rocks_id', prefix='rocks')
+pops_factory = CallbackData('pops_id', prefix='pops')
+country_factory = CallbackData('country_id', prefix='country')
+bot = TeleBot(constant.API_TOKEN)
+markup = types.ReplyKeyboardMarkup(row_width=2)
+btn1 = types.KeyboardButton('/KPOP')
+btn2 = types.KeyboardButton('/ROCK')
+btn3 = types.KeyboardButton('/POP_MUSIC')
+btn4 = types.KeyboardButton('/COUNTRY')
+markup.add(btn1, btn2, btn3, btn4)
+now = datetime.datetime.now()
+
+hideBoard = types.ReplyKeyboardRemove()
+
+
+def greetings():
+    if now.hour < constant.NOON:
+        return "Good morning"
+    elif constant.NOON <= now.hour < constant.EVENING:
+        return "Good afternoon"
+    else:
+        return "Good evening"
+
+
+@bot.message_handler(commands=['start'])
+def send_welcome(message):
+    user = message.from_user.first_name
+    bot.reply_to(message, f"{greetings()} {user} {constant.HEART_EMOJI},\n"
+                          f"Welcome to Songs bot!\n"
+                          f"Ask me any songs I will find it for you \n"
+                          f"{constant.HINT_EMOJI} Type /choose + your song\n"
+                          f"{constant.HINT_EMOJI}  Type /help if you need my guides")
+
+
+@bot.message_handler(commands=['help'])
+def message_handler(message):
+    if dbAlchemy.len_items(message.chat.id) != 0:
+        users_song = dbAlchemy.get_items(message.chat.id)
+        songs = '\n'.join(constant.STAR_EMOJI + str(e) for e in users_song)
+        bot.send_message(message.chat.id, f"Here are your favorites :\n"
+                                          f"{songs}")
+    else:
+        bot.send_message(message.chat.id, "You are new here")
+    bot.send_message(message.chat.id, "Choose your genre of music?", reply_markup=markup)
+
+
+@bot.message_handler(commands=['choose'])
+def send_welcome(message):
+    if not message.text[7:].strip():
+        bot.send_message(message.chat.id, "Please type /choose followed by the song")
+    else:
+        return ContinueHandling()
+
+
+@bot.message_handler(commands=['choose'])
+def start2(message: types.Message):
+    song = message.text.split(' ', 1)[1]  # get the title /or singer after the command /choose
+    videos_search = VideosSearch(song, limit=10, language='en', region='US')
+    data = videos_search.result(mode=ResultMode.json)
+    d1 = json.loads(data)
+    bot.send_message(message.chat.id, "Here is your song")
+    song_id = d1["result"][0]["title"]  # extract the title and singer from json
+    dbAlchemy.add_item(message.chat.id, message.from_user.first_name, song_id)
+    link = d1["result"][0]["link"]  # extract the link
```

Comment:
Hide details of interaction with youtube in another layer. 
Make this entrypoint easy to read.